packages/engine/src/xr/XRState.ts

```diff
@@ -161,4 +161,5 @@ export const isMobileXRHeadset =
       navigator.userAgent.includes('VR') ||
       navigator.userAgent.includes('AR') ||
       navigator.userAgent.includes('Reality') ||
-      navigator.userAgent.includes('Wolvic')
+      navigator.userAgent.includes('Wolvic') ||
+      navigator.userAgent.includes('Mobile')
```

Comment:
isMobileXRHeadset should only be true for mobile headsets (ie non Desktop-VR), not headsets AND mobile devices.

Will this trigger for any non-headset mobile devices?

Reply:
Ah understood. Removing.